app/models/payment_confirmation.rb

```diff
@@ -25,7 +25,7 @@ def ingest
 
       if errors.empty?
         payroll_run.claims.each do |claim|
-          ClaimMailer.payment_confirmation(claim, payment_date_timestamp).deliver_later
+          PaymentMailer.confirmation(claim.payment, payment_date_timestamp).deliver_later
```

Comment:
Given that we only use the `claim` to access its `payment`, does it make sense to loop through `payroll_run.payments` above instead of the claims?

Reply:
Yep - it's in the final commit.

Reply:
👍